calnex/cmd/cert.go

```diff
@@ -17,9 +17,13 @@ limitations under the License.
 package cmd
 
 import (
+	"errors"
+	"net"
 	"os"
+	"time"
 
 	"github.com/facebook/time/calnex/api"
+	c "github.com/facebook/time/calnex/cert"
```

Comment:
Let's move the content of the `cert()` `Run: func(cmd *cobra.Command, args []string) {` this way we will not have to rename the import.